modules/grafana/files/vhost.conf

```diff
@@ -8,5 +12,11 @@ server {
 
   location /app/dashboards/ {
     alias /etc/grafana/dashboards/;
+
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header Host $http_host;
+    proxy_redirect off;
+
+    proxy_pass http://grafana.dev.gov.uk-proxy;
```

Comment:
Can we remove the reference to dev here and just call it `grafana-proxy` or something similar?